eth/contracts/PancakeStableSwap/PancakeStableSwapTwoPoolDeployer.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import "@openzeppelin-4.5.0/contracts/access/Ownable.sol";
+import "./PancakeStableSwapTwoPool.sol";
+
+contract PancakeStableSwapTwoPoolDeployer is Ownable {
+    uint256 public constant N_COINS = 2;
+
+    /**
+     * @notice constructor
+     */
+    constructor() {}
+
+    // returns sorted token addresses, used to handle return values from pairs sorted in this order
+    function sortTokens(address tokenA, address tokenB) internal pure returns (address token0, address token1) {
+        require(tokenA != tokenB, "IDENTICAL_ADDRESSES");
+        (token0, token1) = tokenA < tokenB ? (tokenA, tokenB) : (tokenB, tokenA);
+    }
+
+    /**
+     * @notice createSwapPair
+     * @param _tokenA: Addresses of ERC20 conracts .
+     * @param _tokenB: Addresses of ERC20 conracts .
+     * @param _A: Amplification coefficient multiplied by n * (n - 1)
```

Comment:
1. Do we have a task or know already how we set `_A`, `_fee`, _`admin_fee`, `_admin` and `_LP`?
2. Can we please try to improve imported contract naming practices, e.g.
* `_A` -> `_amplifcationCoefficient`
* `_fee` -> `_exchangesFee` 
* `_LP` -> `_luquidityPool` 
The alternative is to leave this naming as it is, and create a facade/adapter function with proper naming in `eth` folder, and leave those imported files as per originals.

Reply:
these are included here https://cardinal-cryptography.atlassian.net/browse/MOST-88